tests/providers/openlineage/utils/test_utils.py

```diff
@@ -482,3 +484,151 @@ def test_get_task_groups_details_nested():
 
 def test_get_task_groups_details_no_task_groups():
     assert _get_task_groups_details(DAG("test_dag", start_date=datetime.datetime(2024, 6, 1))) == {}
+
+
+@patch("airflow.providers.openlineage.conf.custom_run_facets", return_value=set())
+def test_get_custom_facets_with_no_function_definition(mock_custom_facet_funcs):
+    sample_ti = TaskInstance(
+        task=EmptyOperator(task_id="test-task", dag=DAG("test-dag")),
```

Comment:
@anandhimurali  I believe adding something like this to all places where you create DAG in tests would solve the problem with the last CI check failing and would make it all green😄 
```suggestion
        task=EmptyOperator(task_id="test-task", dag=DAG("test-dag", start_date=datetime.datetime(2024, 7, 1))),
```

Reply:
Thanks again. Addressed in 8c404b93bab1624e8ec893e1fd6b2ff86682a773.